.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "salt/journal-cms/api-dummy"]
```

Comment:
I just committed https://github.com/elifesciences/builder/commit/139e0c73d20ea5122c8f2a137d06195e9fb73208 which means after updating the builder to the latest version you will be able to refer also to all files in `api-dummy-formula` inside this one (removing the need to import it).